fuzz/client.c

```diff
@@ -50,8 +50,6 @@ int FuzzerInitialize(int *argc, char ***argv)
     ERR_get_state();
     CRYPTO_free_ex_index(0, -1);
     idx = SSL_get_ex_data_X509_STORE_CTX_idx();
-    RAND_add("", 1, ENTROPY_NEEDED);
-    RAND_status();
```

Comment:
Please note that the intention of calling RAND_add() and RAND_status() here is to make sure that that the RNG is initialized and that all global variables are set to a specific state that doesn't change anymore when something asks for random bytes or the status.